src/wcmCommon.c

```diff
@@ -860,13 +852,38 @@ void wcmSendEvents(WacomDevicePtr priv, const WacomDeviceState* ds)
 		y = priv->oldState.y;
 	}
 
+	if (!IsPad(priv)) { /* pad doesn't post x/y */
+		wcmAxisSet(&axes, WACOM_AXIS_X, x);
+		wcmAxisSet(&axes, WACOM_AXIS_Y, y);
+		wcmAxisSet(&axes, WACOM_AXIS_PRESSURE, z);
+	}
+
 	/* use tx and ty to report stripx and stripy */
```

Comment:
I've moved the comment down and now assigning `tx`/`ty` for the DBG statement. This whole function could do with a refresh but I'll leave that as a follow-up patch since it's functional change as opposed to just moving things around.